source/user-manual/files-backup/wazuh-agent.rst

```diff
@@ -0,0 +1,118 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: Learn how to keep a backup of key files of your Wazuh agent installation.
+  
+Wazuh agent
+===========
+
+To create a backup of your Wazuh agent installation follow these steps.
+
+.. note::
+
+   You need root user privileges to execute the commands below.
+
+Preparing the backup
+--------------------
+
+#. On the agent machine you're doing the back up for, run the following commands to create the destination folder where to store the files. These commands use date and time references for the folder name to keep files separated from old backups you might have.
+
+   .. tabs::
+
+      .. group-tab:: Linux
+
+         .. code-block:: console
+
+            # bkp_folder=~/wazuh_files_backup/$(date +%F_%H:%M)
+            # mkdir -p $bkp_folder && echo $bkp_folder
+
+      .. group-tab:: Windows
+
+         .. code-block:: doscon
+
+            > set datetime=%date%-%time%
+            > set datetime=%datetime: =_%
+            > set datetime=%datetime:/=-%
+            > set datetime=%datetime::=_%
+            > set datetime=%datetime:.=_%
+            > set bkp_folder=%userprofile%\wazuh_files_backup\%datetime%
+            > mkdir %bkp_folder% && echo %bkp_folder%
+      
+      .. group-tab:: macOS
+
+         .. code-block:: console
+
+            # bkp_folder=~/wazuh_files_backup/$(date +%F_%H:%M)
+            # mkdir -p $bkp_folder && echo $bkp_folder
+
+Backing up a Wazuh agent
+------------------------
+
+#. Backup Wazuh agent data, certificate and configuration files.
+
+   .. tabs::
+
+      .. group-tab:: Linux
+
+         .. code-block:: console
+
+            # rsync -aREz \
+            /var/ossec/etc/client.keys \
+            /var/ossec/etc/ossec.conf \
+            /var/ossec/etc/internal_options.conf \
+            /var/ossec/etc/local_internal_options.conf \
+            /var/ossec/etc/*.pem \
+            /var/ossec/logs/ \
+            /var/ossec/queue/rids/ $bkp_folder
+
```

Comment:
@s-ocando

Yes, also any custom file is important. But I make the same clarification as before, these files could be in different locations depending on each user. Since the paths of these files are specified in the configuration itself.